src/main/java/emu/grasscutter/command/commands/KickCommand.java

```diff
@@ -8,7 +8,7 @@
 
 import static emu.grasscutter.utils.Language.translate;
 
-@Command(label = "kick", usage = "kick", permission = "server.kick", description = "commands.kick.description")
```

Comment:
Kick is one of those commands where it didn't really make sense to have an `others` subpermission as the whole point of the command is to use on other players. `/restart` actually works identically to using `/kick` on oneself. If `kick` is to gain separate permissions for self vs other usage, `restart` may as well be deprecated.